samples/resource.js

```
@@ -38,18 +38,18 @@ function createEntityType(projectId, displayName, kind) {
     },
   };
 
-  entityTypesClient
-    .createEntityType(createEntityTypeRequest)
-    .then(responses => {
-      console.log(`Created ${responses[0].name} entity type`);
-    })
-    .catch(err => {
-      console.error('Failed to create size entity type:', err);
-    });
+  try {
```

just get rid of the try/catch and let it throw